hal/network/ncp_client/quectel/quectel_ncp_client.cpp

```diff
@@ -245,7 +245,15 @@ int QuectelNcpClient::initParser(Stream* stream) {
                 ::sscanf(atResponse, "+CGREG: %u,\"%x\",\"%x\",%u", &val[0], &val[1], &val[2], &val[3]));
         }
         CHECK_TRUE(r >= 1, SYSTEM_ERROR_AT_RESPONSE_UNEXPECTED);
+
+        bool prev_reg_status = self->psd_.registered();
```

Comment:
Nit: mix of snake_case and camelCase. Same in https://github.com/particle-iot/device-os/pull/2174/files#diff-7cf4c0920642e9913b8848733dacef6fR294 https://github.com/particle-iot/device-os/pull/2174/files#diff-ece868c5f58dddb291c6b739de0da632R244 https://github.com/particle-iot/device-os/pull/2174/files#diff-ece868c5f58dddb291c6b739de0da632R288

Reply:
Can you also comment on why we only do this within PSD URCs and not CSD?

Reply:
I forgot to ask - this was an open question for me too. Was thinking that CGREG is the one that would eventually connect to cloud, but it's useful to have CREG as well. thanks for pointing out.